directory-menu@torchipeppo/files/directory-menu@torchipeppo/applet.js

```diff
@@ -163,6 +164,7 @@ class CassettoneApplet extends Applet.IconApplet {
             "x": x,
             "y": y,
             "orientation": o,
+            "terminal_cmd": this.terminal_cmd,
```

Comment:
[3/3] The callback function  beginning at line 170 or 172 is the one that actually takes care of executing the action clicked on the menu, as per the comments on the Python file. The callback is executed after the Python process terminates, with the "response" argument being whatever the process wrote to stdout.